app/templates/direct-award/save-search.html

```diff
@@ -40,61 +62,63 @@
           {% endwith %}
         </div>
 
-        <div class="dmspeak">
-          <h2 class="heading-xmedium">Choose where to save your search</h2>
-        </div>
-        {% if save_search_selection_error %}
-        <p class="validation-message" id="error-save-search-selection">
-          {{ save_search_selection_error }}
-        </p>
-        {% endif %}
+       
         
         <fieldset>
           
+          
+
           <div{% if save_search_selection_error %} class="validation-wrapper"{% endif %}>
           
-          <div class="multiple-choice" data-target="new-search">
-            <input type="radio" name="save_search_selection" value="new_search" id="project-new"
-              {% if request.form.save_search_selection == 'new_search' or not projects %}checked="checked"{% endif %} 
-              required="required">
-            <label for="project-new">Create a new saved search</label>
-          </div>
-          <div class="panel panel-border-narrow js-hidden" id="new-search">
-            {% if form.name.errors %}
-            <div class="validation-wrapper">
+              {% if save_search_selection_error %}
+              <p class="validation-message" id="error-save-search-selection">
+                {{ save_search_selection_error }}
+              </p>
               {% endif %}
-              <div class="question" id="{{ form.name.name }}">
-                {{ form.name.label(class="question-heading") }}
-                <p>Name your saved search. A reference number or short description of what you want to buy makes a good name.</p>
-                <p class="hint">
-                  100 characters maximum
-                </p>
-                {% if form.name.errors %}
-                <p class="validation-message" id="error-name-textbox">
-                  {% for error in form.name.errors %}{{ error }}{% endfor %}
-                </p>
-                {% endif %}
-                {{ form.name(class="text-box") }}
+
+              {% if projects %}
+
+              
+
+              {% for project in projects %}
+
+              <div class="multiple-choice">
+                <input type="radio" name="save_search_selection" value="{{ project.id }}" id="project-{{ project.id }}"{% if request.form.save_search_selection == project.id %}checked="checked"{% endif %}>
+                <label for="project-{{ project.id }}">{{ project.name or "No Project Name" }}</label>
```

Comment:
Although it should be impossible, it might be better to use `project.id` rather than "No Project Name", to cover the theoretical case where two projects don't have names and would then be indistinguishable.

But we definitely shouldn't be able to have projects without names...

Reply:
Agree, perhaps "Untitled {{id}}" is vaguely traditional